### PR TITLE
docs: getting started

### PR DESCRIPTION
**Pull request by andy-haynes**
This PR adds a `Getting Started` section to the project README with instructions on how to deploy an application using the monorepo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,59 @@ A gateway server could also be distributed as a binary to be run locally on a us
...
+using the specified view library. The process is largely the same regardless of the template chosen:
+
+1. Run `pnpm i && pnpm build` at the monorepo project root. 
+2. Configure the `nearDeployConfig` field in `package.json`:
+   1. `application` is developer-defined and will be used as part of the URL (names should match `[a-z_-]+`)
+   2. `deployerAccount` is the account paying for bundle storage and calling smart contract methods (must match `DEPLOYER_ACCOUNT.near` referenced above)
+   3. `filestoreContract` is the hosted storage contract (`pfs1.testnet` on testnet and `v1.chainui.near` on mainnet, or deployed and configured separately)
+3. Add components, content, and/or NPM dependencies to the application.
+4. Run `pnpm run deploy` to build the project bundle and deploy the application on chain.
+5. Load the application at `http://ec2-54-185-81-147.us-west-2.compute.amazonaws.com/FILE_CONTRACT.near/DEPLOYER_ACCOUNT.near/APPLICATION-NAME` (with `FILE_CONTRACT` being `pfs1.testnet` or `v1.chainui.near`)
```
Comment (andy-haynes) [inline README.md:77]: This requires a forthcoming `cdn` update 😅

File: README.md [hunk 2]
```diff
@@ -37,3 +37,59 @@ A gateway server could also be distributed as a binary to be run locally on a us
...
+of whether the application is already deployed. Once the deployment is live, the files from the previous deployment
+are deleted and storage is refunded as part of the deployment script.
+
+### New Projects
+
+New projects may use the `react` or `vue` packages; demo packages preconfigured to produce and deploy applications
+using the specified view library. The process is largely the same regardless of the template chosen:
+
+1. Run `pnpm i && pnpm build` at the monorepo project root. 
+2. Configure the `nearDeployConfig` field in `package.json`:
```
Comment (mpeterdev) [inline README.md:71]: add step in between 1&2 to cd into `packages/react` or `packages/vue`

File: README.md [hunk 3]
```diff
@@ -37,3 +37,59 @@ A gateway server could also be distributed as a binary to be run locally on a us
...
+   1. `application` is developer-defined and will be used as part of the URL (names should match `[a-z_-]+`)
+   2. `deployerAccount` is the account paying for bundle storage and calling smart contract methods (must match `DEPLOYER_ACCOUNT.near` referenced above)
```
Comment (mpeterdev) [inline README.md:73]:
```suggestion
   2. `deployerAccount` is your account that pays for bundle storage and calls smart contract methods. This corresponds to the keystore created above (must match `DEPLOYER_ACCOUNT.near`)
```

File: README.md [hunk 4]
```diff
@@ -37,3 +37,59 @@ A gateway server could also be distributed as a binary to be run locally on a us
...
+   3. `filestoreContract` is the hosted storage contract (`pfs1.testnet` on testnet and `v1.chainui.near` on mainnet, or deployed and configured separately)
```
Comment (mpeterdev) [inline README.md:74]:
```suggestion
   3. `filestoreContract` is the chain-hosted-ui contract (`pfs1.testnet` on testnet and `v1.chainui.near` on mainnet, or deployed and configured separately)
```

File: README.md [hunk 5]
```diff
@@ -37,3 +37,59 @@ A gateway server could also be distributed as a binary to be run locally on a us
...
+3. Add components, content, and/or NPM dependencies to the application.
+4. Run `pnpm run deploy` to build the project bundle and deploy the application on chain.
```
Comment (mpeterdev) [inline README.md:76]:
```suggestion
4. Run `pnpm run deploy` to build the project bundle and deploy the application on chain. You will be presented with the estimated cost to approve before executing the deploy
```

**Review by mpeterdev — CHANGES_REQUESTED** [review]

**Review by mpeterdev — APPROVED** [review]

**Pull request merged**